ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/vlan-interfaces.md

```diff
@@ -500,6 +549,115 @@
         isis_passive: <bool>
         isis_metric: <int>
         isis_network_point_to_point: <bool>
+        isis_authentication:
+
+          # Authentication settings for level-1 and level-2. 'both' takes precedence over 'level_1' and 'level_2' settings.
+          both:
+
+            # Configure authentication key type. Default key_id is 0.
+            key_type: <str; "0" | "7" | "8a">
+
+            # Password string.
```

Comment:
update the description- 
`key_type` is required to configure this setting.

Reply:
Fixed.